economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+
+\begin{equation}
+    S(\infty) = S_0  +  S_0 \cdot \int_0^{5} I_{max}(t)\, dt  +  S_0 \cdot \int_0^{\infty} I(t)\, dt
+\end{equation}
+\\
+Note that we use 5 years as a provisional duration for Phase 1 in order to calculate the maximum possible issuance. In practice, the issuance will be lower, and hence the switching of phases ($T_{1\rightarrow2}$) will likely occur at $t$ > 5 years – see equation 12. Note also that the boundaries of the second integral are $0$ and $\infty$, rather than $5$ and $\infty$, because the issuance curve in Phase 2 is computed independently of Phase 1. In Phase 2, the decaying issuance at a time $t$ is a function of $I_{max}$, since it commences at this figure.
+ 
+\begin{equation}
+    I(t) = I_{max} \cdot 2^{-\frac{t}{T_{1/2}}}
+\end{equation}
+\begin{equation}
+    3.885 \cdot S_0  = S_0 + S_0 \cdot \int_0^{5} I_{max}(t)\, dt + S_0 \cdot \int_0^{\infty} I_{max} \cdot 2^{-\frac{t}{T_{1/2}}} \, dt ,\\
+\end{equation}
+\begin{equation}
+    3.885 = 1 + 5 \cdot  I_{max} + I_{max} \cdot \left[\frac{-2^{(1-\frac{t}{2})}}{\ln{2}}\right]_0^\infty .\\
+\end{equation}
+
+\begin{equation}
+    2.885 = I_{max} \cdot (5 + \frac{2}{\ln{2}})
+\end{equation}
+
+\begin{equation}
+   I_{max} = \frac{2.885}{7.885} = 36.6\%
+\end{equation}
+\\
+We multiply by $S_0$ and convert $I_{max}$ into the absolute number of tokens minted per period.
+
+\begin{equation}
+    \label{eq:abs}
+   \frac{I_{max} \cdot S_0}{365} = 1,002,509\:\:new \:tokens \:per \:period
+\end{equation}
+
+Hence, the two-phase schedule sees a maximum of 1,002,509 tokens minted every period until there are 2,829,579,800 billion tokens in the circulating supply. At this point the protocol will switch to Phase 2, in which a further 1,055,810,282 tokens will be minted until t = $\infty$.
+
+
+\section{Staking protocol}
+
+\subsection{Subsidy size and commitment time}
+
+\subsubsection{Sub-stakes}
+
+In order to be eligible to answer user requests, earn fees, and receive subsidies, a service-provider must commit to servicing the network for some period of time by locking collateral (staking). Service-providers specify a stake unlocking time in the future $t_1$, where at time $t$ the minimum duration $t_1 - t$ may not be fewer than $D_{\min}$ = 31 periods, but may be any greater number of periods .
+\\\\
+The NuCypher protocol allows service-providers to partition their stake into sub-stakes, up to a maximum of 30. A sub-stake has a unique remaining duration $D$ – the number of periods until it unlocks and the tokens become freely withdrawable. The sub-stake size $\theta$ is the number of tokens locked. It is possible to extend (but not reduce) $D$ for a sub-stake, or split a sub-stake by extending $D$ for part of it. It is also possible to acquire more tokens and increase $\theta$ in any sub-stake.
```

Comment:
To solidify this section, we should indicate the motivation for such a setting i.e. it provides different liquidity needs for different portions of the stake.

Reply:
```suggestion
The NuCypher protocol allows service-providers to partition their stake into sub-stakes, up to a maximum of 30. A sub-stake has a unique remaining duration $D$ – the number of periods until it unlocks and the tokens become freely withdrawable. The sub-stake size $\theta$ is the number of tokens locked. It is possible to extend (but not reduce) $D$ for a sub-stake, or split a sub-stake by extending $D$ for part of it. It is also possible to acquire more tokens and increase $\theta$ in any sub-stake. This mechanism enables more granular planning for future liquidity needs. 
```